fv_be/app/models/dictionary.py

```diff
@@ -0,0 +1,125 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+# FirstVoices
+from .base import BaseModel
+from .category import Category
+from .part_of_speech import PartOfSpeech
+
+
+class Note(BaseModel):
+    """Model for notes associated to each dictionary entry."""
+
+    # from fv:notes,fv:general_note, fv:cultural_note, fv:literal_translation, fv-word:notes, fv-phrase:notes
+    text = models.TextField()
+
+    def __str__(self):
+        return self.text
+
+
+class Acknowledgment(BaseModel):
+    """Model for acknowledgments associated to each dictionary entry."""
+
+    # from fv:acknowledgments, fv:source, fv:reference, fv-word:acknowledgement, fv-phrase:acknowledgement
+    text = models.TextField()
+
+    def __str__(self):
+        return self.text
+
+
+class Translation(BaseModel):
+    """Model for translations associated to each dictionary entry."""
+
+    # Choices for Language
+    ENGLISH_ENUM_KEY = "EN"
+    FRENCH_ENUM_KEY = "FR"
+    LANGUAGE_CHOICES = [
+        (ENGLISH_ENUM_KEY, _("English")),
+        (FRENCH_ENUM_KEY, _("French")),
+    ]
+
+    # Fields
+    translation = models.CharField(max_length=200)
+    language = models.CharField(
+        max_length=2, choices=LANGUAGE_CHOICES, default=ENGLISH_ENUM_KEY
+    )
+    # from fv-word:part_of_speech
+    part_of_speech = models.ManyToManyField(PartOfSpeech)
+    # todo: more representative name for the following attribute ?
+    parent = models.ForeignKey("DictionaryEntry", on_delete=models.CASCADE)
+
+    def __str__(self):
+        return _("Translation in %(language)s: %(translation)s.") % {
+            "language": self.language,
+            "translation": self.translation,
+        }
+
+
+class AlternateSpelling(BaseModel):
+    """Model for alternate spellings associated to each dictionary entry."""
```

Comment:
Note: I don't think Alternate Spellings are set up on our V2 edit forms right now. We can still include them, but it will be a later feature to add them to the edit forms. (It would also be awesome to make text in this field part of the site search at that time.)

Reply:
Okay, I believe we could remove it for now and then add it back later when we're adding stuff to it.

Reply:
There is existing data in the fields though, so does that mean we should add it now since it will have to be part of migration? Or is the idea that we would get back to this and add it before we actually migrate any data? (@sarahfirstvoices ?)

Reply:
Oh okay, If there is data in the fields, I would suggest we should add the model now and then add that data as a part of the migration, even if it is not setup on the edit forms.